apps/ledger-live-mobile/src/components/ServicesWidget/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { protectContext } from "@ledgerhq/live-common/platform/providers/ProtectProvider/index";
+import { ProtectStateNumberEnum } from "@ledgerhq/live-common/platform/providers/ProtectProvider/types";
+import React, { memo, useContext } from "react";
+import { useTranslation } from "react-i18next";
+import Svg, { LinearGradient, Defs, Rect, Stop } from "react-native-svg";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import NewProtectState from "./Protect/NewProtectState";
+import ConfirmIdentityProtectState from "./Protect/ConfirmIdentityProtectState";
+import AddPaymentProtectState from "./Protect/AddPaymentProtectState";
+import SubscriptionCanceledProtectState from "./Protect/SubscriptionCanceledProtectState";
+import PaymentRejectedProtectState from "./Protect/PaymentRejectedProtectState";
+import ActiveProtectState from "./Protect/ActiveProtectState";
+import { ServicesConfig } from "./types";
+
+const SvgGradient = () => (
+  <Svg width="100%" height="8px">
+    <Defs>
+      <LinearGradient
+        id="protectGradient"
+        x1="0%"
+        y1="0%"
+        x2="100%"
+        y2="0%"
+        gradientUnits="userSpaceOnUse"
+      >
+        <Stop offset="0%" stopOpacity={1} stopColor="hsla(40, 90%, 69%, 1)" />
+        <Stop offset="100%" stopOpacity={1} stopColor="hsla(40, 96%, 81%, 1)" />
+      </LinearGradient>
+    </Defs>
+    <Rect x="0" y="0" width="100%" height="100%" fill="url(#protectGradient)" />
+  </Svg>
+);
+
+const statesKeys: Record<ProtectStateNumberEnum, string> = {
+  [ProtectStateNumberEnum.NEW]: "new",
+  [ProtectStateNumberEnum.CONFIRM_IDENTITY]: "confirmIdentity",
+  [ProtectStateNumberEnum.ADD_PAYMENT]: "addPayment",
+  [ProtectStateNumberEnum.PAYMENT_REJECTED]: "paymentRejected",
+  [ProtectStateNumberEnum.SUBSCRIPTION_CANCELED]: "subscriptionCanceled",
+  [ProtectStateNumberEnum.ACTIVE]: "active",
+};
+
+const statesComponents: Record<
+  ProtectStateNumberEnum,
+  React.FunctionComponent<{ params: Record<string, string> }> & {
+    StatusTag: React.FunctionComponent<Record<string, never>>;
+  }
+> = {
+  [ProtectStateNumberEnum.NEW]: NewProtectState,
+  [ProtectStateNumberEnum.CONFIRM_IDENTITY]: ConfirmIdentityProtectState,
+  [ProtectStateNumberEnum.ADD_PAYMENT]: AddPaymentProtectState,
+  [ProtectStateNumberEnum.PAYMENT_REJECTED]: PaymentRejectedProtectState,
+  [ProtectStateNumberEnum.SUBSCRIPTION_CANCELED]:
+    SubscriptionCanceledProtectState,
+  [ProtectStateNumberEnum.ACTIVE]: ActiveProtectState,
+};
+
+function ServicesWidget() {
+  const { t } = useTranslation();
+  const servicesConfig: ServicesConfig | null = useFeature(
+    "protectServicesMobile",
+  );
+
+  const { enabled, params } = servicesConfig || {};
+
+  const {
+    state: { protectState },
+  } = useContext(protectContext);
+
+  const ProtectStateComponent = statesComponents[protectState];
+
+  return enabled && params?.managerStatesData ? (
+    <>
+      <Text mt={12} variant="body" fontSize="20px">
```

Comment:
Please don't use fixed fontSize="20px", use the right variant.
If the figma is not referencing a variant, then the design is bad and you should use the closest value :safety_pin: